parabolDocker/.env_test

```diff
@@ -0,0 +1,37 @@
+ATLASSIAN_CLIENT_ID='key_ATLASSIAN_CLIENT_ID'
```

Comment:
At the end, if you would like to move to k8s, I think you need to redesign it to pass trough environment variable.
we could re-discuss it in the future

Reply:
The app as its written today will read its configuration from environment variables. Any variables found in `.env` will override those of the environment.

The `.env.test` file is used as the default environment when running our automated cypress tests.

Reply:
Finlay, because you use env var, I will update the way I source this information.
All env will be directly in the Dockerfile (default) / docker-compose (modify values if need),
by this way could be run it on k8s without any modification.